### PR TITLE
DevEx - Removing API Smoketests

### PR DESCRIPTION
**Pull request by codyseibert**
removing the api smoke tests which have been failing on node14 for years without anyone knowing

### CODE REVIEW

**Review by jasonfu9966 — APPROVED** [review]
nice catch! strange catch. what are the circumstances around this that made it fly under the radar for so long? was the result unreported?

**Review by jimlerza — APPROVED** [review]

**Comment by jimlerza** [comment]
> nice catch! strange catch. what are the circumstances around this that made it fly under the radar for so long? was the result unreported?

The axios call was always returning an error, but for some reason the node process itself was returning a 0, which made it "pass" in circleci. As of node 16, the process returns a 1, which results in a failure in circleci.

File: .circleci/config.yml [hunk 1]
```diff
@@ -484,6 +484,7 @@ jobs:
           name: Admin User Setup
           command: |
             if [ "${CIRCLE_BRANCH}" != "prod" ]; then
+              . ./scripts/load-environment-from-secrets.sh
```
Comment (mrinsin) [inline .circleci/config.yml:487]: Do we still need to source the vars from secrets? I know this was part of our debugging effort but I may have lost context at some point!

Comment (codyseibert) [reply .circleci/config.yml:487]: yeah, I think at one point I mentioned we might be able to remove those, but I'm pretty sure that fixed an issue where it kept trying to create the admin user with admin@example.com when the secrets expected it to create ustcadmin@example.com.  Without this change the build creates the wrong admin user name.

Comment (mrinsin) [reply .circleci/config.yml:487]: Oh that's right - thank you for explaining it to me again!

**Review by mrinsin — APPROVED** [review]

File: scripts/data-import/judge/bulkImportJudgeUsers.helpers.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const init = async (csvFile, outputMap) => {
           } else {
             const services = await getServices();
             endpoint = `${
-              services[`gateway_api_${process.env.DEPLOYING_COLOR}`]
+              services[`gateway_api_${process.env.CURRENT_COLOR}`]
```
Comment (rachelschneiderman) [inline scripts/data-import/judge/bulkImportJudgeUsers.helpers.js]: This is breaking deploys to experimental environments, I believe we want to import judge users into the `deploying_color` so we can run the smoketests with judges populated. Was this an accidental commit during debugging?

Comment (rachelschneiderman) [reply scripts/data-import/judge/bulkImportJudgeUsers.helpers.js]: Since Cody is out this week, I made this change 😄

**Review by TomElliottFlexion — APPROVED** [review]

**Review by mmarcotte — APPROVED** [review]

**Comment by mmarcotte** [comment]
Merging before tests pass (as they will be re-run) and this change only affects CircleCI's `config.yml`

**Pull request merged**